network/phonebook.go

```diff
@@ -163,14 +175,9 @@ func (mp *MultiPhonebook) GetAddresses(n int) []string {
 	return out
 }
 
-// AddPhonebook adds a Phonebook if it is new
-func (mp *MultiPhonebook) AddPhonebook(p Phonebook) {
+// AddOrUpdatePhonebook adds or updates Phonebook in Pnonebook map
+func (mp *MultiPhonebook) AddOrUpdatePhonebook(name string, p Phonebook) {
```

Comment:
Could you please give the variable "name" into some more descriptive name ? maybe bootstrapNetworkName ?

Reply:
done